src/Migration/Pdk/ProductSettingsMigration.php

```diff
@@ -365,8 +365,6 @@ private function migrateProduct(WC_Product $wcProduct): void
 
         $this->debug(sprintf('Settings for product %s migrated', $wcProduct->get_id()));
         $this->markMigrated($wcProduct);
```

Comment:
dit gaat niet goed, dit wordt nu niet opgeslagen

Reply:
markMigrated doet `$object->update_meta_data($migratedKey, array_merge($executedMigrations, [$this->getVersion()]));` dat slaat toch meteen op?

Reply:
Zo niet dan mea culpa

Reply:
dat slaat niet op

Reply:
Inderdaad ik zie het.